config/default-cluster/pipeline-role.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
```

Comment:
can we have :

* one resource per file
* file always named `{metadata.name}-{kind.ToLowerCase}.yaml`